awx/main/models/inventory.py

```diff
@@ -1211,12 +1225,21 @@ def get_ec2_group_by_choices(cls):
             ('ami_id', _('Image ID')),
             ('availability_zone', _('Availability Zone')),
             ('aws_account', _('Account')),
+            # These should have been added, but plugins do not support them
```

Comment:
it seems like all of these that you've commented out have just been moved out of the inventory plugin that we are using (relative to the old script). So we can just remove these commented out values.